webui/react/src/utils/task.ts

```diff
@@ -182,7 +182,7 @@ export const isExperimentTask = (task: Type.AnyTask): task is Type.ExperimentTas
 export const isTaskKillable = (task: Type.AnyTask | Type.ExperimentItem): boolean => {
   return (
     killableRunStates.includes(task.state as Type.RunState) ||
-    killableCommandStates.includes(task.state as Type.CommandState)
```

Comment:
What's the reason for this change?

Reply:
I got type error cuz `task.state` doesnt satisfies `killableCommandStates`
we can do `map().some()` alternatively

Reply:
You should fix the type of `killableCommandStates` then, it should be an `Array<CommandState>`.

Reply:
fixed